doc/update023.md

```diff
@@ -0,0 +1,31 @@
+**The Hush coin's codebase is ready for Season 4 Activation in 3P node**
+
+- Please update your Hush codebase to [7f7876d](https://github.com/MyHush/hush3/tree/7f7876d2084daebc89b1914c870fa407c4fbef4e)
+- Build it
+
+```bash
+cd ~/hush3
+./zcutil/build.sh
```

Comment:
since you write so detailed instruction :) - it's better to use more than single core to not wait very long